cmd/rook/rook/rook.go

```diff
@@ -63,3 +75,16 @@ func SetupLogging() {
 func NewTableWriter(buffer io.Writer) *tabwriter.Writer {
 	return tabwriter.NewWriter(buffer, outputMinWidth, outputTabWidth, outputPadding, outputPadChar, 0)
 }
+
+func setFlagsFromEnv(flags *pflag.FlagSet) error {
```

Comment:
Would have to parameterize the prefix but that seems reasonable (i.e. this uses ROOK_... and the other use ROOKD_...)